classes/ClaytonTestClass1.cls

```diff
@@ -0,0 +1,953 @@
+public class ClaytonTestClass1 {
+    public ClaytonTestClass1(){
+        List<Case> cList = [Select Id,CaseNumber, AccountId, 
+                            ContactId, Contact.AccountId From Case];
+        
+        for(Case c : cList){
+            if(!String.isBlank(c.CaseNumber)){
+                if(c.ContactId != null){
+                    if(c.AccountId != null){
+                        if(c.Contact.AccountId != null){
+                            if(c.AccountId != c.Contact.AccountId){
```

Comment:
Nesting too many `IF` statements affects readability and makes maintenance more difficult. Consider refactoring to simplify the flow of this code block.